### DISCUSSION TITLE
Should we use gRPC or http for nitro <-> triton

### DISCUSSION THREAD

**Discussion by tikikun** [janhq/nitro#34] [Announcements]
HTTP PROS:
- No more dependencies since drogon can be the client itself
- Very lean code can leverage compression for speed up if needed

gRPC PROS:
- Better for high throughput
- Add a lot of dependencies to manage
- Nvidia triton client codebase is not supporting multiple system

Refer to this issue
https://github.com/janhq/nitro/issues/33

**Comment by hiro-v** [comment]
Decision: Use HTTP to Triton
